tests-legacy/PrestaShopBundle/Controller/ControllerTest.php

```diff
@@ -97,7 +97,6 @@ public function getControllersClasses()
             array('AdminCarriersController'),
             array('AdminStatusesController'),
             array('AdminZonesController'),
-            array('AdminCurrenciesController'),
```

Comment:
You can add the symfony test in https://github.com/PrestaShop/PrestaShop/blob/develop/tests-legacy/Integration/PrestaShopBundle/Controller/Admin/SurvivalTest.php 😉